src/main/kotlin/org/jitsi/jibri/util/ProcessWrapper.kt

```diff
@@ -88,8 +88,10 @@ abstract class ProcessWrapper<out StatusType>(
     fun waitFor(timeout: Long, unit: TimeUnit): Boolean = process.waitFor(timeout, unit)
 
     fun stop() {
-        tail.stop()
-        tee.stop()
```

Comment:
these lines were the cause of the bug that got introduced with the original version of this fix. because shutdown wasn't called but we stopped these tasks, tee never read EOF which means it never closed the inputstreams that others were reading from, which means that when we try and wait for the logStream task to finish (the processLoggerTask?.get() above, which waits until it reads the last bit from the stream) we blocked forever.